### PR TITLE
Fix @types/json-schema version in dashboard/package.json

### PR DESCRIPTION
**Pull request by jessehu**
The @types/json-schema version in dashboard/package.json is out of date. So make it consistent with the version in yarn.lock.

### CODE REVIEW

**Comment by andresmgot** [comment]
I think you need to run `yarn upgrade ...` and upload both the package.json and yarn.lock changes for this to work.

**Comment by jessehu** [comment]
I tried 'yarn upgrade' and it didn't update package.json. So I think there is no need for 'yarn upgrade', but updating package.json separately.

File: dashboard/yarn.lock [hunk 1]
```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
-  integrity sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==
```
Comment (andresmgot) [inline dashboard/yarn.lock]: I think  you need to upgrade `yarn`, this has deleted all the integrity info

Comment (jessehu) [reply dashboard/yarn.lock]: I used yarn 1.7.0. Now using v1.17.3 and the integrity info is kept. Please review again.

**Pull request merged**